### PR TITLE
Added Contribution Guidelines

### PR DESCRIPTION
**Pull request by Shaishav0507**
Issue Number : #299 

Hey @siv2r 
I have added contribution guidelines and made some minor changes.
Please check and update it.
Thanks for letting me work on this project.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,10 +28,101 @@ After a doctor completes this process, this platform provides two main features.
...
+
+**14.** Add appropriate title and description to your pull request explaining your changes and efforts done.
+
+**15.** Click on `Create Pull Request`.
+
+**16.** Sit back patiently and relax while the project maintainers review your PR. Please understand, at times the time taken to review a PR can vary from a few hours to a few days.
+
+<p align="center"><img src="https://media.tenor.com/images/b562ddcfb131e962f9dfa01bd32a30d1/tenor.gif" width=30%></p>
+
+
```
Comment (siv2r) [inline README.md:118]: can you remove this change? (from line 31 to 118)
There is already a document present in the doc folder explaining this

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,124 @@
...
+https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
+
+Community Impact Guidelines were inspired by [Mozilla's code of conduct
+enforcement ladder](https://github.com/mozilla/diversity).
+
+[homepage]: https://www.contributor-covenant.org
+
+For answers to common questions about this code of conduct, see the FAQ at
+https://www.contributor-covenant.org/faq. Translations are available at
+https://www.contributor-covenant.org/translations.
```
Comment (siv2r) [inline CODE_OF_CONDUCT.md:124]: Can you remove this `CODE_OF_CONDUCT.md`?

**Review by siv2r — CHANGES_REQUESTED** [review]

**Comment by Shaishav0507** [comment]
Done

**Comment by siv2r** [comment]
Thanks for the contribution!!

**Pull request merged**